src/components/atoms/Button.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 
-import './button.css';
+import './button.css'
 
 interface Props {
   className?: string;
   title: string;
   onClick: () => void;
+  disabled?: boolean;
 }
 
-const Button: React.FC<Props> = ({ className = '', title, onClick }) => {
+const Button: React.FC<Props> = ({ className = '', title, onClick, disabled = false }) => {
   return (
-    <div className={`${className} button`} onClick={onClick}>
+    <div className={`${className} ${disabled && "disabled"}`} onClick={onClick}>
```

Comment:
Tu devrais gérer le disabled depuis le composant et non depuis le parent car ça veut dire qu'a chaque fois tu vas changer devoir écrire le code dans le parents...

fais un truc comme ça 

```suggestion
    <div className={`${className} ${disabled && "disabled"}`} onClick={() => !disabled && onClick()}>>
```